src/core/Akka/Delivery/ConsumerController.cs

```diff
@@ -0,0 +1,275 @@
+// -----------------------------------------------------------------------
+//  <copyright file="ConsumerController.cs" company="Akka.NET Project">
+//      Copyright (C) 2009-2023 Lightbend Inc. <http://www.lightbend.com>
+//      Copyright (C) 2013-2023 .NET Foundation <https://github.com/akkadotnet/akka.net>
+//  </copyright>
+// -----------------------------------------------------------------------
+#nullable enable
```

Comment:
Enabled all new APIs to support `#nullable` going forward. Didn't change project defaults - but for new APIs going forward I think it'd be best if we started doing this.